THIRD-PARTY-NOTICES

```diff
@@ -1372,4 +1372,77 @@ necessary.  Here is a sample; alter the names:
   <signature of Ty Coon>, 1 April 1990
   Ty Coon, President of Vice
 
-That's all there is to it!
\ No newline at end of file
+That's all there is to it!
+
+----------------------------------------------------------------------
+
+License notice for the DiffPlex NuGet package (https://www.nuget.org/packages/DiffPlex)
+------------------------------------------------------------
```

Comment:
There isn't a copyright notice for the project, but we do need to include the license.